src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -22,6 +22,7 @@ internal class SqlQueryGenerator : DefaultExpressionVisitor<SearchOptions, objec
         private readonly bool _isHistorySearch;
         private int _tableExpressionCounter = -1;
         private SqlRootExpression _rootExpression;
+        private const int MaxIncludedItems = 100;
```

Comment:
Is there a shared config this value can be set in instead of hard coded in the code?

Reply:
short answer is yes. however there is an issue. opened an issue to keep track...

Reply:
ok. please add the issue#